src/field/pokemon.ts

```diff
@@ -2156,21 +2167,29 @@ export default abstract class Pokemon extends Phaser.GameObjects.Container {
     const evasionMultiplier = new Utils.NumberHolder(1);
     applyStatMultiplierAbAttrs(StatMultiplierAbAttr, target, Stat.EVA, evasionMultiplier);
 
-    return accuracyMultiplier.value / evasionMultiplier.value;
+    accuracyMultiplier.value /= evasionMultiplier.value;
+
+    return accuracyMultiplier.value;
   }
 
   /**
-  * Apply the results of a move to this pokemon
-  * @param {Pokemon} source The pokemon using the move
-  * @param {PokemonMove} battlerMove The move being used
-  * @returns {HitResult} The result of the attack
-  */
-  apply(source: Pokemon, move: Move): HitResult {
-    let result: HitResult;
+   * Calculates the damage of an attack made by another Pokemon against this Pokemon
+   * @param source {@linkcode Pokemon} the attacking Pokemon
+   * @param move {@linkcode Pokemon} the move used in the attack
+   * @param ignoreAbility `boolean` if `true`, ignores this Pokemon's defensive ability effects
+   * @param isCritical `boolean` if `true`, calculates damage for a critical hit.
+   * @param simulated `boolean` if `true`, suppresses changes to game state during the calculation.
+   * @returns a {@linkcode DamageCalculationResult} object with four fields:
+   * - `move`: {@linkcode Moves} the identifier for the attacking move.
```

Comment:
It doesn't actually return `move` (any more?) though.

Reply:
```suggestion
   * @returns a {@linkcode DamageCalculationResult} object with three fields:
```
yeah, this was from an old version of the code